src/gt4py/cartesian/frontend/gtscript_frontend.py

```diff
@@ -1889,7 +1889,9 @@ def resolve_external_symbols(
                         "{}.{}".format(value._gtscript_["qualified_name"], item_name): item_value
                         for item_name, item_value in value._gtscript_["nonlocals"].items()
                     }
-                    resolved_values_list.extend(nested_inlined_values.items())
+                    resolved_values_list.extend(  # noqa[B038] #editing a loop's mutable iterable (probably intended here)
```

Comment:
flake8 complains here, not sure if my comment is correct and it's really intended.